unicodetools/src/main/java/org/unicode/tools/emoji/Emoji.java

```diff
@@ -40,8 +40,49 @@
 import com.ibm.icu.util.ULocale;
 import com.ibm.icu.util.VersionInfo;
 
+/**
+ * You need a command line variable to generate either the beta version (not yet released) or the abbreviated version (see below).
+ * <p>Example: -Demoji-beta
```

Comment:
The emoji-beta option is already mentioned on https://github.com/unicode-org/unicodetools/blob/main/docs/emoji/index.md
Wouldn't it be better to update that and point from here to there?

Note that there are several docs pages under emoji/ : https://github.com/unicode-org/unicodetools/tree/main/docs/emoji